scripts/modify_proxy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
can we have a folder called `cd-scripts` and put this one inside, as well as the entire appspec-scripts folder?

Reply:
also can we add a short comment saying that this script is to map port to the latest service?